Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -668,6 +670,9 @@ private void InitializeActions()
                     break;
                 }
 
+            if (uiInputModule != null)
+                uiInputModule.actionsAsset = m_Actions;
```

Comment:
This seems to create an initialization ordering issue between the two systems. If the UI module happens to initialize first, it will go do its thing based on the data it has only to then see the stuff it has set up get stomped when PlayerInput initializes.

Reply:
It should still work correctly, though - it just means that it does some unnecessary work. I guess I can change the execution order on PlayerInput to -1, that should fix it, right?

Reply:
Yup, could be that's good enough for now.